statsdbinterface/function_cache.py

```diff
@@ -1,6 +1,12 @@
 import time
 from threading import Thread, Lock
 import atexit
+try:
+    import config
+    cache_enabled = config.CACHE
```

Comment:
Well, then the module's design would have to be adapted to support the `init_app` pattern that is popular among Flask extensions. But anyway, as I intend to replace it, I don't care very much.